wherehows-web/app/typings/global-plugin.d.ts

```diff
@@ -4,4 +4,11 @@ import Ember from 'ember';
 declare global {
   // eslint-disable-next-line typescript/no-empty-interface, typescript/interface-name-prefix
   interface Array<T> extends Ember.ArrayPrototypeExtensions<T> {}
+
+  interface IHandlerInfo {
```

Comment:
Confused why this interface for tracking is here?

Reply:
Not sure where to put it  🆘

Reply:
This is defined as an [abstract class](https://github.com/tildeio/router.js/blob/73d6a7f5ca6fba3cfff8460245fb5a3ceef8a2b5/lib/router/handler-info.ts#L324) in the extracted routing lib for emberjs, routerjs. Can we 
- preferred: `yarn install router_js` and use those types directly?
- in the worst case: add a comment here explaining the reference and the decision

Reply:
Ohh!! I didn't know the existence of that. Thanks!!

Reply:
Yea, haven’t tried this before but I think those types are public